newIDE/app/src/ProjectCreation/LocalPathFinder.js

```diff
@@ -19,13 +19,30 @@ const findEmptyPath = (basePath: string) => {
   return path.join(basePath, folderName);
 };
 
-export const findEmptyPathInDefaultFolder = (electronApp: any): string => {
+/**
+ * Returns the default workspace folder from the Documents folder or the Home folder
+ */
+export const findDefaultFolder = (electronApp: any): string => {
   let documentsPath = '';
   try {
     documentsPath = electronApp.getPath('documents');
   } catch (ex) {
     // A user may not have the Documents folder defined on Windows.
     documentsPath = electronApp.getPath('home');
   }
-  return findEmptyPath(path.join(documentsPath, 'GDevelop projects'));
+  return path.join(documentsPath, 'GDevelop projects');
+};
+
+/**
+ * Returns the current workspace with a generated project name
+ */
+export const findEmptyPathInWorkspaceFolder = (
+  electronApp: any,
+  workspaceFolder: string | null
+): string => {
+  const folder =
+    workspaceFolder === null || workspaceFolder === undefined
```

Comment:
No need to check for undefined because the typing tells that this can be only null or a string ;) 
```suggestion
    workspaceFolder === null
```